### PR TITLE
Implemented editing for comments.

### PR DESCRIPTION
**Pull request by Kiyooooko**
Now we can edit comment and go to product-details page to see more about product:
Screenshot 2021-02-11 at 15 45 57
Screenshot 2021-02-11 at 15 46 10
Screenshot 2021-02-11 at 15 46 17

Also see user comment on user-details page: 
Screenshot 2021-02-11 at 15 46 33

And on product-datils page:
Screenshot 2021-02-11 at 15 53 48

### CODE REVIEW

**Review by AndriyLymych — DISMISSED** [review]

File: src/components/forms/comment-form/comment-form.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    },
+    onSubmit: (data) => {
+      if (isEdit) {
+        dispatch(
+          updateComment({
+            id,
+            comment: {
+              text: data.text,
+              show: data.show,
+              user: comment.user._id,
```
Comment (moran711) [inline src/components/forms/comment-form/comment-form.js:58]: Why you need it here if you update text?

File: src/components/forms/comment-form/comment-form.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+              product: comment.product._id
```
Comment (moran711) [inline src/components/forms/comment-form/comment-form.js:59]: same

File: src/components/forms/comment-form/comment-form.js [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+      value: values.show,
+      checked: values.show,
+      color: 'primary',
+      label: config.labels.comment.show,
+      handler: () => setFieldValue('show', !values.show)
+    }
+  ];
+
+  function handleProductClick() {
+    history.push(`/product/${comment.product._id}`);
```
Comment (moran711) [inline src/components/forms/comment-form/comment-form.js:80]: please get `/product` from configs

File: src/pages/comments/comments.js [hunk 4]
```diff
@@ -64,6 +65,9 @@ const Comments = () => {
       deleteHandler={() => {
         commentDeleteHandler(comment._id);
       }}
+      editHandler={() => {
+        dispatch(push(`/comments/${comment._id}`));
```
Comment (moran711) [inline src/pages/comments/comments.js:69]: same

**Review by Haytham1337 — APPROVED** [review]

**Review by AndriyLymych — APPROVED** [review]

**Review by Haytham1337 — APPROVED** [review]

**Review by moran711 — APPROVED** [review]

**Review by AndriyLymych — APPROVED** [review]

**Review by jul4ajka — APPROVED** [review]

**Pull request merged**